pontoon/base/templates/users.html

```diff
@@ -13,7 +13,7 @@
       <tr>
         <th>Rank</th>
         <th>Contributor</th>
-        <th>Translations</th>
+        <th>Translations <span>* without unchanged</span></th>
```

Comment:
`sup` would probably be a better tag to use here. 

Also I think adding something like a footer with a complete sentence like "Translation counts do not include unchanged translations." would be better, maybe with a `sup` tag next to the header that links to the footnote.